x-pack/plugins/secops/public/components/resize_handle/index.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import { fromEvent, Observable, Subscription } from 'rxjs';
+import { mergeMap, takeUntil } from 'rxjs/operators';
+import styled, { injectGlobal } from 'styled-components';
+
+interface OnResizeParams {
+  delta: number;
+  id: string;
+}
+
+export type OnResize = ({ delta, id }: OnResizeParams) => void;
+
+export const resizeCursorStyle = 'col-resize';
+export const globalResizeCursorClassName = 'global-resize-cursor';
+
+// tslint:disable-next-line:no-unused-expression
+injectGlobal`
+.${globalResizeCursorClassName} {
+  * {
+      cursor: ${resizeCursorStyle};
+      &: hover {
+        cursor: ${resizeCursorStyle};
+      }
+  }
+}
+`;
+
+interface Props {
+  /** the `onResize` callback will be invoked with this id */
+  id: string;
+  /** The resizeable content to render */
+  render: (isResizing: boolean) => React.ReactNode;
+  /** a (styled) resize handle */
+  handle: React.ReactNode;
+  /** optionally provide a height style ResizeHandleContainer */
+  height?: string;
+  /** invoked when the handle is resized */
+  onResize: OnResize;
+}
+
+const ResizeHandleContainer = styled.div<{ height?: string }>`
+  cursor: ${resizeCursorStyle};
+  ${({ height }) => (height != null ? `height: ${height}` : '')}
```

Comment:
That's interesting that you are returning an empty string. Is that like a `null` where Styled won't add the property? It won't add height as an empty string right?

Reply:
It becomes a no-op (there is no height specified)